.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,5 +1,5 @@
 <!--
-    Thank you for your interest in contributing to Nautobot! Please note
+    Thank you for your interest in contributing to Golden Config! Please note
```

Comment:
Something is off because PR templates are not working

Reply:
@whitej6 trying to get this remark resolved and I was wondering how you managed to test this and if your comment stands true still. Any pointers?

Reply:
Marking this as resolved for the moment. If the issue still stands after merging, we'll address it in a separate PR.